lib/resty/redis.lua

```diff
@@ -268,8 +267,9 @@ local function _gen_req(args)
         end
     end
 
-    -- it is faster to do string concatenation on the Lua land
-    return concat(req)
+    -- it is much faster to do string concatenation on the C land
+    -- in some real world (large number of string in Lua)
```

Comment:
"in some real world" -> "in real world"
"large number of string in Lua" -> "large number of strings in the Lua VM"